### PR TITLE
[NEW] Avatar update using URL

### LINKED ISSUE #754: [NEW] Update user avatar via URL
From the profile view, allow to the user update its avatar via URL. The idea is simple: have a action to update the avatar and an input to the user insert that URL, after all call the [client.setAvatar(avatarUrl: String)](https://github.com/RocketChat/Rocket.Chat.Kotlin.SDK/blob/99b4ce6c2e79a7c6f7a4d2a9e3a8b2b719b94720/core/src/main/kotlin/chat/rocket/core/internal/rest/User.kt#L117).

### PR DESCRIPTION
**Pull request by SyamSundarKirubakaran**
@RocketChat/android

Closes #754 

screen shot 2018-02-19 at 6 17 37 pm

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt [hunk 1]
```diff
@@ -116,22 +118,30 @@ class ProfileFragment : Fragment(), ProfileView, ActionMode.Callback {
...
 
     private fun listenToChanges() {
-        Observables.combineLatest(text_name.asObservable(), text_username.asObservable(), text_email.asObservable()).subscribe({ t ->
-                    if (t.first.toString() != currentName || t.second.toString() != currentUsername || t.third.toString() != currentEmail) {
-                        startActionMode()
-                    } else {
-                        finishActionMode()
-                    }
-                })
+        Observables.combineLatest(text_name.asObservable(),text_username.asObservable()).subscribe({ t ->
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt]: As I told you, take a look of how to observe more than three observables here.

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt (hunk 1 above)
Comment (SyamSundarKirubakaran) [reply app/src/main/java/chat/rocket/android/profile/ui/ProfileFragment.kt]: Ok @filipedelimabrito  I'll look into it and making changes accordingly..

**Comment by rafaelks** [comment]
@filipedelimabrito Is this PR ready to be merged? Some changes were made.

**Review by philipbrito — APPROVED** [review]

**Comment by philipbrito** [comment]
Thanks for this PR @SyamSundarKirubakaran!

File: app/src/main/res/layout/fragment_profile.xml [hunk 2]
```diff
@@ -46,6 +46,13 @@
             android:drawableStart="@drawable/ic_email_black_24dp"
             android:hint="@string/msg_email"
             android:inputType="textEmailAddress" />
+        <EditText
+            android:id="@+id/text_avatar_url"
+            style="@style/EditText.Profile"
```
Comment (aniketsingh03) [inline app/src/main/res/layout/fragment_profile.xml:51]: Shouldn't this be `@style/Profile.EditText`, the one you defined doesn't exist!

Comment (SyamSundarKirubakaran) [reply app/src/main/res/layout/fragment_profile.xml:51]: Yes, `@style` was changed after me forking it and giving this PR.. 
@luciofm has fixed with a commit.. Do a pull..

**Pull request merged**